hs/src/Reach/Connector/ETH_Solidity.hs

```diff
@@ -951,6 +951,8 @@ solCom = \case
   DL_Let _ (DLV_Let _ dv) (DLE_PrimOp _ GET_COMPANION []) -> do
     addMemVar dv
     solLargeArg dv $ mdaToMaybeLA T_Contract Nothing
+  DL_Let _ (DLV_Eff) (DLE_GetUntrackedFunds {}) ->
+    return ""
```

Comment:
Why this?

Reply:
An impossible? Did you add a test case?

Reply:
I discovered this while tweaking your duoswap PR to use `.api_`. When the shared checks and bindings are duplicated into PAY_EXPR, the result of `getUntrackedFunds` is unused and updated to `DLV_Eff`. Maybe, the purity of `getUntrackedFunds` should be handled specially during optimization so it doesn't make it this far?